detectron2/layers/deform_conv.py

```diff
@@ -46,7 +47,9 @@ def forward(
         ctx.bufs_ = [input.new_empty(0), input.new_empty(0)]  # columns, ones
 
         if not input.is_cuda:
-            raise NotImplementedError("Deformable Conv is not supported on CPUs!")
+            return deform_conv2d(
+                input, offset, weight, stride=stride, padding=padding, dilation=dilation
+            )
```

Comment:
Input arguments `groups`, `deformable_groups` are not used.
Either pass them to torchvision, or assert their values if not supported by torchvision.

Reply:
The `groups` arguments is supported because torchvision infers it from weights' shape. Added the assertion for `deformable_groups`.